kube/src/mock_tests.rs

```diff
@@ -61,6 +61,7 @@ async fn timeout_after_1s(handle: tokio::task::JoinHandle<()>) {
 /// Scenarios we test for in ApiServerVerifier above
 enum Scenario {
     PaginatedList,
+    #[allow(dead_code)]
     RadioSilence,
```

Comment:
Does nothing use this variant then?

Reply:
it's matched on in a select, but we don't instantiate it atm, was planning on maybe using it in a test, as it's a common mock base scenario and it felt weird to not have it

Reply:
Fair, I'd maybe mention that in a comment.

Reply:
minor comment added in https://github.com/kube-rs/kube/pull/1436/commits/f49d4edb77e6c71f8f6fa4eebe644c547198bef4